apps/web/src/lib/quotation-html/template.html

```
@@ -0,0 +1,717 @@
+<meta charset="utf-8" />
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)